src/shared/graphql/request/carRequest.tsx

```diff
@@ -0,0 +1,289 @@
+import {
+  ADD_CAR,
+  ADD_FAVORITE_CAR,
+  FIND_CARS,
+  GET_CAR,
+  GET_FAVORITE_CAR,
+  REMOVE_FAVORITE_CAR,
+} from '../query/carQuery';
+import { Cars, User_Cars } from '../__generate__/generated';
+import { useLazyQuery, useMutation, useQuery } from '@apollo/client';
+
+import { IFormInput } from '../../../pages/ViewCreateCar/ViewCreateCar';
+
+export const useGetCar = (carId: number, userId: number | undefined) => {
+  const variables = {
+    where: {
+      id: {
+        _eq: carId,
+      },
+    },
+    userCarsWhere: {
+      _and: [
+        {
+          car_id: {
+            _eq: carId,
+          },
+          user_id: {
+            _eq: userId,
+          },
+        },
+      ],
+    },
+  };
+  return useQuery<{ cars: Cars[]; user_cars: User_Cars[] }>(GET_CAR, {
+    variables: {
+      ...variables,
+    },
+    fetchPolicy: 'network-only',
+  });
+};
+
+export const useFindCar = () => {
+  const [getCars, result] = useLazyQuery(FIND_CARS);
```

Comment:
I don't think you need to use lazy query, really at all. There are very few times I've found that it's necessary.